analysis/benchmarking-functions/R/quant_info_table.R

```diff
@@ -53,10 +53,7 @@ quant_info_table <- function(data_dir, tools, samples){
   all_quant_info <- dplyr::bind_rows(quant_info_list) %>%
     dplyr::mutate(alevin_alignment = tidyr::replace_na(alevin_alignment, "not_alevin"),
                   alevin_resolution = tidyr::replace_na(alevin_resolution, "not_alevin"),
-                  usa_mode = ifelse(alevin_resolution == "cr" & 
-                                      index_type == "splici", TRUE, FALSE),
-                  intron_mode = ifelse(tool != "cellranger" & 
-                                         usa_mode == FALSE & 
-                                         index_type == "splici", TRUE, FALSE))
+                  usa_mode = index_type == "splici" & alevin_resolution == "cr" &  
```

Comment:
I had an extra `&` in there... would get caught quickly, but I noticed it now.
```suggestion
                  usa_mode = index_type == "splici" & alevin_resolution == "cr" 
```